### ISSUE TITLE
API 500 Error even when creating a zone succeeds

### ISSUE THREAD

**Issue by vikas027** [PowerDNS/pdns#8216]
### Environment
- Operating system:  Ubuntu 19.04
- Software version: PowerDNS 4.1.6
- Software source: Debian official repos

### Steps to reproduce
1. Install and configure powerdns
```bash
$ apt-get -y install pdns-server pdns-backend-mysql
```

2. Configure backend and enable API
I am following the [official](https://doc.powerdns.com/md/httpapi/README/) doc.

3. Create a Zone
```bash

$ curl POST -v -H 'X-API-Key: changeme' http://127.0.0.1:8082/api/v1/servers/localhost/zones \
> --data '{
>   "name": "example.org.",
>   "kind": "Native",
>   "masters": [],
>   "nameservers": ["ns1.example.org."]
> }'
* Expire in 0 ms for 6 (transfer 0x5557597084f0)
<redacted>
* Expire in 7 ms for 1 (transfer 0x5557597084f0)
* Could not resolve host: POST
* Expire in 9 ms for 1 (transfer 0x5557597084f0)
* Closing connection 0
curl: (6) Could not resolve host: POST
* Expire in 0 ms for 6 (transfer 0x5557597084f0)
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Expire in 200 ms for 4 (transfer 0x5557597084f0)
* Connected to 127.0.0.1 (127.0.0.1) port 8082 (#1)
> POST /api/v1/servers/localhost/zones HTTP/1.1
> Host: 127.0.0.1:8082
> User-Agent: curl/7.64.0
> Accept: */*
> X-API-Key: changeme
> Content-Length: 104
> Content-Type: application/x-www-form-urlencoded
> 
* upload completely sent off: 104 out of 104 bytes
< HTTP/1.1 500 Internal Server Error
< Connection: close
< Content-Length: 21
< Content-Type: text/plain; charset=utf-8
< Server: PowerDNS/4.1.6
< 
* Closing connection 1
Internal Server Error$ 

$ curl -sH 'X-API-Key: changeme' http://127.0.0.1:8082/api/v1/servers/localhost/zones | jq .
[
  {
    "account": "",
    "dnssec": false,
    "id": "example.org.",
    "kind": "Native",
    "last_check": 0,
    "masters": [],
    "name": "example.org.",
    "notified_serial": 0,
    "serial": 0,
    "url": "/api/v1/servers/localhost/zones/example.org."
  }
]
$
```

### Expected behaviour
Creation a zone should not return error 500 on success.

### Actual behaviour
Curl errors out with 500 code

### Other information
Backend is MySQL

**Comment by Habbie** [comment]
```
* upload completely sent off: 104 out of 104 bytes
< HTTP/1.1 201 Created
< Access-Control-Allow-Origin: *
< Connection: close
< Content-Length: 687
< Content-Security-Policy: default-src 'self'; style-src 'self' 'unsafe-inline'
< Content-Type: application/json
< Server: PowerDNS/0.0.HEAD.g646d566d7.dirty
< X-Content-Type-Options: nosniff
< X-Frame-Options: deny
< X-Permitted-Cross-Domain-Policies: none
< X-Xss-Protection: 1; mode=block
< 
* Closing connection 1
{"account": "", "api_rectify": false, "dnssec": false, "id": "example.org.", "kind": "Native", "last_check": 0, "masters": [], "name": "example.org.", "notified_serial": 0, "nsec3narrow": false, "nsec3param": "", "rrsets": [{"comments": [], "name": "example.org.", "records": [{"content": "a.misconfigured.powerdns.server. hostmaster.example.org. 2019081901 10800 3600 604800 3600", "disabled": false}], "ttl": 3600, "type": "SOA"}, {"comments": [], "name": "example.org.", "records": [{"content": "ns1.example.org.", "disabled": false}], "ttl": 3600, "type": "NS"}], "serial": 2019081901, "soa_edit": "", "soa_edit_api": "DEFAULT", "url": "/api/v1/servers/localhost/zones/example.org."}
```

I cannot reproduce your issue here. Can you please
* enable `query-logging`
* try with a newer version (4.1.13 or 4.2.0-rc3)
? Thank you.

(In 4.2.0, you can set `webserver-loglevel=detailed` for additional information.)

**Comment by vikas027** [comment]
Hey @Habbie ,

After a lot of hit and trials, I found that in v4.1.6 API call was failing in the between means it was able to populate the database. I am not sure why, but even command `pdnsutil create-zone` failed with error `Unknown column change_date in field list`. Although, [this](https://github.com/PowerDNS/pdns/blob/rel/auth-4.1.x/modules/gmysqlbackend/schema.mysql.sql) schema says that it is for 4.1.X version but it did not work in 4.1.6

Everything started working normally when I installed version 4.1.1.
